src/main/java/com/ogp/configurator/ConnectionLossException.java

```diff
@@ -0,0 +1,8 @@
+package com.ogp.configurator;
+
+/**
+ * @author Anton Kharenko
+ */
+public class ConnectionLossException extends RuntimeException {
+	//TODO: Java Doc and implementation
+}
```

Comment:
May be will be better to declare ConnectionLossException  explicitly (extend from Exception)?
It will force developer to threate ConnectionLoss situation.